src/components/container/EhrNewsFeedEventDetail/EhrNewsFeedEventDetail.tsx

```diff
@@ -62,8 +62,8 @@ function NewsFeedDetailTitle(props: { event: EhrNewsFeedEventModel }) {
     let title = handler.getDetailTitle ? handler.getDetailTitle(props.event) : undefined;
     if (!title) { return null; }
 
-    let subtitle = `${capitalizeForLocale(formatDateForLocale(props.event.Date, "EEEE, MMMM do, y h:mm a"))} • ${props.event.Patient.RecordAuthority}`
-    if (formatDateForLocale(props.event.Date, "h:mm a") == "12:00 AM") {
+    let subtitle = `${getDayAndDateAndTimeString(props.event.Date)} • ${props.event.Patient.RecordAuthority}`
+    if (getTimeOfDayString(props.event.Date) === "") {
```

Comment:
The helper returns an empty string for a time of 00:00:00 so this replaces the 'if' check above.